winui/Circular-Charts/Selection.md

```diff
@@ -69,9 +78,15 @@ Circular chart allows to select single or multiple segment\series using [Type](h
 
 SfCircularChart chart = new SfCircularChart();
 . . .
-ChartSelectionBehavior selection = new ChartSelectionBehavior();
-selection.Type = SelectionType.MultiPoint;
-chart.Behaviors.Add(selection);
+PieSeries series = new PieSeries();
+DataPointSelectionBehavior selection = new DataPointSelectionBehavior()
+{
+	SelectionBrush = new SolidColorBrush(Colors.BlueViolet),
+    Type = SelectionType.Multiple
+};
+
+series.SelectionBehavior = selection;
+chart.Series.Add(series);
```

Comment:
@naveenkumar-sanjeevirayan : Inselection feature -> SelectedIndex, SelectedIndexes property not explined. explain that too in all chart.